exporter/opensearchexporter/README.md

```diff
@@ -0,0 +1,66 @@
+# OpenSearch Exporter
+
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [devel]   |
+| Supported pipeline types | traces    |
+| Distributions            | [contrib] |
+
+This exporter supports sending OpenTelemetry signals as documents to [OpenSearch](https://www.opensearch.org).
+
+The documents are sent using [observability catalog](https://github.com/opensearch-project/opensearch-catalog/tree/main/schema/observability) schema.
+
+## Configuration options
+
+- `endpoints`: List of OpenSearch URLs. If endpoints is missing, the
+  OPENSEARCH_URL environment variable will be used.
```

Comment:
This is not typically how configuration defaults are handled in the collector.  As environment variables can be interpolated into configuration files it is usually left to the user to determine what variables to use.  This will be complicated by the fact that this takes a list.  How would the value of this variable be parsed to obtain a list of URLs?  What will the exporter do with a list of endpoints?  Does it export simultaneously to all endpoints? Load-balance between them in some way?

Reply:
Fair point re: environment variable.

As for `endpoints` property, it is passed to the OpenSearch Go client. It maintains it's own connection pool.